src/console/Command.cpp

```diff
@@ -0,0 +1,91 @@
+#include "console/Command.hpp"
+#include "storm/Error.hpp"
+
+int32_t ValidateFileName(const char* filename);
```

Comment:
I think this was missed from my earlier feedback: for private-to-file functions like this one, just declare and implement the function in one go, at the top of the file (before it is used).